Dockerfile.alpine

```diff
@@ -17,12 +17,16 @@ RUN apk add --no-cache bzip2 ca-certificates coreutils libstdc++ tini w3m &&\
   apk del bzip2 w3m
 
 # create user, group, and set permissions
-RUN adduser -u 503 -g 503 -h ${TS_DIRECTORY} -D teamspeak &&\
-  mkdir /data &&\
-  chown -R teamspeak:teamspeak ${TS_DIRECTORY} /data
+RUN  echo "**** create teamspeak user and make our folders ****" && \
```

Comment:
The issue with this approach is that a user has to build the image in order to set the UID/GID and it doesn't get set at runtime.